test_plans/kernels_parameters.asciidoc

```diff
@@ -0,0 +1,56 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for parameter passing to kernels
+
+This is a test plan for rules for parameter passing to kernels defined in https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:kernel.parameter.passing[4.12.4. Rules for parameter passing to kernels].
+
+Other legal types of parameters covered in other groups of tests.
+
+`sampled_image_accessor` and `unsampled_image_accessor` are expected to be tested in future for passing to kernels in tests for these accessors.
+
+== Testing scope
+
+This plan does not include negative tests.
+
+=== Device coverage
+
+All the tests described below are performed only on the default device that is selected on the CTS command line.
+
+=== Type coverage
+
+Tests are performed for types:
+
+* `marray<T, 5>`
+* `vec<T, 4>`
+* `T[5]`
+* `class S with a non-static member variable of type T`
+* `class K non-virtual base class S`
```

Comment:
The following types are also allowed to be kernel arguments:

* `id`
* `range`

Reply:
Added.